mplview/core.py

```diff
@@ -234,8 +236,15 @@ def __init__(self,
 
         self.next_cid = 0
 
+        mpl_ver = distutils.version.LooseVersion(matplotlib.__version__)
+        mpl_2_ver = distutils.version.LooseVersion("2")
+
+        fig_axes_kwargs = {}
+        axcolor_name = "facecolor" if mpl_ver >= mpl_2_ver else "axisbg"
```

Comment:
Checked that this works correctly for dev releases as well. Though it is not really relevant any more.